### ISSUE TITLE
pedestrians zig-zag in network with walkingareas

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#5302]
when there are no walkingareas, a single connector is build which connects all adjacent edges at a node (IntermodalNetwork.h:155). 
This obscures the differences between staying on the same sidewalk and changing the side of the road which causes zig-zag on a straight sequence of edges with sidewalks on both sides

**Comment by namdre** [comment]
I recall we discussed this and decided on the current code for sake of simplicity...

**Comment by behrisch** [comment]
Yes, and I don't see a good way to fix this unless we decide a little bias in walking "forward" (meaning in the direction of the edge) would be acceptable. Then we could add just an epsilon to the cost of the backward edges.

**Comment by namdre** [comment]
I suppose a bias wouldn't hurt since this type of simulation has reduced accuracy in pedestrian movement anyway. 

An alternative would be to create an NxN connection matrix at each junction with air-distance length for each of the connections.
